core/unix/coredump.c

```diff
@@ -27,13 +27,16 @@
  */
 
 #include <elf.h>
+#include <stddef.h>
 #include <sys/mman.h>
 #include "../globals.h"
 
 #include "../hashtable.h"
 #include "../os_shared.h"
 #include "../synch.h"
-#include "dr_tools.h"
+#include "../utils.h"
+#include "../lib/globals_api.h"
+#include "../lib/globals_shared.h"
```

Comment:
This will not be needed after https://github.com/DynamoRIO/dynamorio/pull/7073.

Reply:
Ack.